svs.tex

```diff
@@ -39,96 +39,90 @@ \subsection{Objectives}
-                \item 825 visits / pointing $(u, g, r, i, z, y)$ = (56, 80, 184, 184, 160, 160)
-                \item 100 deg$^2$ in single contiguous region overlapping an LSST DDF; nominal LSST WFD dither pattern (might need intra-night translational offsets and rotator angles to simulate expected LSST distribution)
-                \item Steadily build integrated exposure during the survey
-                \item During a 30 day period, each pointing would receive average of 25 visits on each night
-                %\item Representative DRP data products: Source, Object, ForcedSource, DiaSource, DiaObject, DiaForcedSource
-                %\item Representaitive AP mode: DiaObject, DiaSource, DIAForcedSource
-
-        \end{itemize}
-
-        \item During Next 30 days of Science Validation survey period: increase to 20-year LSST WFD exposure, 1650 visits / pointing, $(u, g, r, i, z, y)$ = (112, 160, 368, 368, 320, 320)
+Rubin Observatory has developed and simulated a Feature Based Scheduler (FBS) configuration for the enhanced baseline SV survey design that could be implemented and is expected to meet the Construction completeness criteria if executed.
 
+As of early June 2025, the forecast of on-sky commissioning activities includes pilot observations for the SV surveys beginning in late June, and sustained SV survey observations running from early July to mid September. The objectives of the SV survey pilot observations are to
+\begin{itemize}
+        \item evaluate the distribution of delivered image quality, including AOS performance, for wide-area survey observations,
+        \item continue optimization of the AOS open loop and closed loop control systems,
+        \item evaluate the overall operational uptime in wide-area survey mode observations similar to the nominal LSST cadence,
+        \item and validate the FBS performance and the observing strategy for the SV surveys.
 \end{itemize}
 
-\textit{Wide Survey}
-
+The enhanced baseline SV survey design includes three survey components that are interleaved as part of a single FBS configuration:
 \begin{itemize}
+        \item a \textbf{Deep Survey} that is optimized for testing deep coadds at the equivalent integrated exposure of the LSST 10-year survey and beyond, achieving a rapid temporal sampling in those fields, and validating the observing strategy for the LSST Deep Drilling Fields (DDFs);
+        \item a \textbf{Wide Survey} that is optimized for testing template generation and Prompt Processing with difference image analysis at data rates that would be expected during the first year of LSST, thereby providing a sustained full-scale test of the Data Facility;
+        \item a \textbf{Template Generation Survey} that is optimized for building wide-area template coverage, including to support target-of-opportunity science cases during the first year of LSST.
+\end{itemize}
```

Comment:
How does the third survey differ from the second survey?  Any survey from which templates are generated should also provide science images?